### PR TITLE
This repo will be archived, project moved to aspnet/AspNetCore

### PR DESCRIPTION
**Pull request by natemcmaster**
As a part of https://github.com/aspnet/AspNetCore/issues/3598, this repository will be made read-only and its contents merged into aspnet/AspNetCore.

cref https://github.com/aspnet/AspNetCore/pull/4273

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
+ASP.NET Core SignalR [Archived]
+===============================
 
-**IMPORTANT**: This repository hosts code and project management for ASP.NET **Core** SignalR, for use in ASP.NET Core applications using `Microsoft.AspNetCore.App`. If you are looking for information on ASP.NET SignalR (used in .NET Framework applications using System.Web and/or Katana), see the https://github.com/SignalR/SignalR repository.
-
-[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/aspnet/SignalR/SignalR-ci)](https://dnceng.visualstudio.com/public/_build/latest?definitionId=26)
-[![NuGet version](https://badge.fury.io/nu/microsoft.aspnetcore.signalr.svg)](https://badge.fury.io/nu/microsoft.aspnetcore.signalr)
-[![npm version](https://badge.fury.io/js/%40aspnet%2Fsignalr.svg)](https://badge.fury.io/js/%40aspnet%2Fsignalr)
-[![Maven Version](https://maven-badges.herokuapp.com/maven-central/com.microsoft.signalr/signalr/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.microsoft.signalr/signalr)
+**This GitHub project has been archived.** Ongoing development on this project can be found in <https://github.com/aspnet/AspNetCore>.
```
Comment (BrennanConroy) [inline README.md]: Can we link to the SignalR folder directly?

Comment (natemcmaster) [reply README.md]: I'd rather avoid broken links if we decide to change our minds on source code layout (it's been known to happen.)

Comment (BrennanConroy) [reply README.md]: Ok 😢

**Review by mikaelm12 — APPROVED** [review]
Bye aspnet/SignalR. Hello aspnet/AspNetCore/src/SignalR 👋 🙁

**Review by analogrelay — APPROVED** [review]

**Pull request merged**